docs/samples-and-tutorials/index.md

```diff
@@ -1,111 +1,122 @@
 
-## Deploying to Containers
+## Deploying to containers
 
 **[Running ASP.NET MVC Applications in Windows Docker Containers](../framework/docker/aspnetmvc.md)**
 
-This tutorial demonstrates how to deploy an existing ASP.NET MVC application in a Windows Docker Container.
-A finished version of the code you'll build is located [in the docs repository on GitHub](https://github.com/dotnet/docs/tree/master/samples/framework/docker/MVCRandomAnswerGenerator).
+This tutorial demonstrates how to deploy an existing ASP.NET MVC app in a Windows Docker Container. The [completed sample](https://github.com/dotnet/docs/tree/master/samples/framework/docker/MVCRandomAnswerGenerator) is available in the dotnet/docs repository on GitHub.
 
 **[Running .NET Framework Console Applications in Windows Containers](../framework/docker/console.md)**
 
-This tutorial demonstrates how to deploy and existing console application in a Windows container. A finished version of the code you'll build is located [in the docs repository on GitHub](https://github.com/dotnet/docs/tree/master/samples/framework/docker/ConsoleRandomAnswerGenerator).
\ No newline at end of file
+This tutorial demonstrates how to deploy an existing console app in a Windows container. The [completed sample](https://github.com/dotnet/docs/tree/master/samples/framework/docker/ConsoleRandomAnswerGenerator) is available in the dotnet/docs repository on GitHub.
+
+## Viewing and downloading samples
+
+Many topics show source code and samples that are available for viewing or download from GitHub. To view a sample, follow the sample link to the sample folder at aspnet/Docs or dotnet/docs on GitHub. To download the code, follow these instructions:
+
+1. Download the repository containing the sample code for your local system by performing one of the following procedures:
+   * [Fork](https://help.github.com/articles/fork-a-repo/) the repository and [clone](https://help.github.com/articles/cloning-a-repository/) the fork to your local system. Forking and cloning permits you to make contributions to the .NET documentation by committing changes to your fork and then creating a pull request for the official docs repository. For more information, see the [.NET Documentation Contributing Guide](https://github.com/dotnet/docs/blob/master/CONTRIBUTING.md) and the [ASP.NET Docs Contributing Guide](https://github.com/aspnet/Docs/blob/master/CONTRIBUTING.md).
+   * Clone the repository locally. Note that if you clone a docs repository directly to your local system that you won't be able to make commits directly against the official repository, so you won't be able to make documentation contributions later. Use the fork and clone procedure previously described if you want to preserve the opportunity to contribute to the documentation later.
+   * Download a ZIP of the repository to your local system. Un-ZIP the compressed archive.
```

Comment:
Consider making this one first, It's simplest and probably most appropriate for most people.